lib/chef/cookbook_loader.rb

```diff
@@ -27,7 +27,7 @@
 
 class Chef
   # This class is used by knife, cheffs and legacy chef-solo modes.  It is not used by the server mode
-  # of chef-client or zolo/zero modes.
```

Comment:
This might have been intentional. Zero + Solo was referred to as zolo.

Reply:
TIL, reverted.